### PR TITLE
Upgrade edX next-residential to Juniper Alpha release

### PR DESCRIPTION
**Pull request by markbreedlove**
Upgrade the next-residential edx app to the Juniper Alpha release.

### CODE REVIEW

**Comment by blarghmatey** [comment]
:+1:

**Comment by markbreedlove** [comment]
@blarghmatey : Thanks for giving this the thumbs up earlier. I have since added one commit that adds that `RUN_MIGRATIONS` flag we were talking about.

File: salt/orchestrate/edx/build_ami.sls [hunk 1]
```diff
@@ -4,6 +4,7 @@
 {% set env_settings = env_dict.environments[ENVIRONMENT] %}
 {% set VPC_NAME = salt.environ.get('VPC_NAME', env_settings.vpc_name) %}
 {% set BUSINESS_UNIT = salt.environ.get('BUSINESS_UNIT', env_settings.business_unit) %}
+{% set RUN_MIGRATIONS = salt.environ.get('RUN_MIGRATIONS') %}
```
Comment (blarghmatey) [inline salt/orchestrate/edx/build_ami.sls]: Should add a default value of `False` just to help clarify that it's a boolean.

Comment (blarghmatey) [reply salt/orchestrate/edx/build_ami.sls]: We should also remove the EDX_VERSION line above as it no longer has any purpose.

Comment (markbreedlove) [reply salt/orchestrate/edx/build_ami.sls]: OK, I've made those changes.

**Comment by blarghmatey** [comment]
:+1:

**Pull request merged**